tests/conftest.py

```diff
@@ -1,13 +1,6 @@
-import pytest
-
-from huntflow_api_client import HuntflowAPI
-from huntflow_api_client.tokens import ApiToken
-
-API_URL = "https://api.huntflow.dev/v2"
-API_TOKEN = ApiToken("mocked token")
-
-
-@pytest.fixture
-def api_client() -> HuntflowAPI:
-    api_client = HuntflowAPI(base_url=API_URL, token=API_TOKEN)
-    return api_client
+pytest_plugins = [
+    "tests.fixtures.api",
+    "tests.fixtures.huntflow",
+    "tests.fixtures.token_proxy",
+    "tests.fixtures.tokens",
```

Comment:
All these fixtures make it difficult to read tests and force you to access private methods

Reply:
I believe that these fixtures help to reduce the template code and make it easier to read tests